vm/chef/cookbooks/espocrm/files/espocrm-utils

```diff
@@ -81,6 +81,7 @@ function change_php_config() {
 #######################################
 function install_app() {
   local -r curl="curl --request POST -c /tmp/cookie -b /tmp/cookie"
+  local -r URL="http://localhost/install/"
   echo "$URL"
   $curl $URL --data "user-lang=en_US&action=step1"
```

Comment:
Can you please try to use the snippet below for all the requests? 

```shell
$curl "${URL}" ...
```

It seems the default document for this directory is indeed index.php, so no need to mention. It seems strange some requests using the index.php and some of them, no.

Reply:
done